devtools/server/actors/replay/telemetry.js

```diff
@@ -1,13 +1,29 @@
 var EXPORTED_SYMBOLS = ["pingTelemetry"];
 
 const { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
+ChromeUtils.defineModuleGetter(
+  this,
+  "ReplayAuth",
+  "resource://devtools/server/actors/replay/auth.js"
+);
 
 function pingTelemetry(source, name, data) {
   const url = Services.prefs.getStringPref("replay.telemetry.url");
   const enabled = Services.prefs.getBoolPref("replay.telemetry.enabled");
 
   if (!enabled || !url) return;
 
+  let user = "unknown";
+  if (ReplayAuth.hasOriginalApiKey()) {
+    user = "api-key";
+  } else {
+    const token = ReplayAuth.getReplayUserToken();
+    if (token) {
+      const tokenInfo = ReplayAuth.tokenInfo(token);
+      user = tokenInfo ? tokenInfo.payload : "invalid-json";
```

Comment:
I don't think anything in here is useful other than `sub` so we could also consider sending that directly.